modules/sqlite3/3.47.2/patches/add_build_file.patch

```diff
@@ -0,0 +1,34 @@
+--- /dev/null
```

Comment:
Did this change? If not, could you replace it with a symlink to the previous version?